stream_in.c

```diff
@@ -264,7 +264,7 @@ int in_get_capture_position(const struct audio_stream_in *stream,
         *time = xin->last_time;
     }
     /* this trace generates lots of messages */
-    /* ALOGD("igcp: %" PRIu64 ", %" PRIu64, xin->read_frames, xin->last_time); */
+    /* LOG_FN_PARAMETERS("igcp: %" PRIu64 ", %" PRIu64, xin->read_frames, xin->last_time); */
```

Comment:
Please remove dead code

Reply:
Thanks for pointing. This fix will be issued as separate patch in separate pull request, as far as it has no relation to output devices.